.env.sample

```diff
@@ -18,3 +18,4 @@ CONTENT_ANALYSIS_BASE_URL=http://localhost:3001
 
 # Leave commented out if not wanted
 # FORCE_2FA=true
+# DISABLE_2FA=true
```

Comment:
We should probably add a comment heading for this envvar that it is a prod-only environment variable that isn't required to be set as part of a developer's `.env` file. Ideally we should capture this in a readme, but in absence of a section for that, this would be good enough.